matrix_sdk/src/event_handler/mod.rs

```diff
@@ -820,4 +841,45 @@ mod test {
             ],
         )
     }
+
+    #[async_test]
+    async fn event_handler_two_syncs() {
```

Comment:
This could do with a better name, so we know that we're testing the notifications here.

Reply:
I added it to have a better test for notifications, but it in a more general way it tests two sequential syncs, that's why I named it like that.

I can change it if you don't like it though.

Reply:
Ah ok, let's leave it as is then.